packages/grid/src/vaadin-grid-mixin.js

```diff
@@ -425,11 +425,13 @@ export const GridMixin = (superClass) =>
             // Prepare the cell for having its intrinsic width measured
             cell.style.width = 'auto';
             cell.style.position = 'absolute';
+            cell.setAttribute('measuring-auto-width', '');
```

Comment:
nit: I wonder if we could avoid toggling this attribute for every cell by toggling it once on one of the parent elements (`this.$.scroller` for example) with selector `[measuring-auto-width] [part~='cell'] > [tabindex]`

Reply:
Yep, it works... I've changed the code accordingly. Thanks

Reply:
Looks like it's still being toggled separately for each auto-width column. Since it causes a layout change, it might affect the performance of a grid with lots of columns. What I meant was adding the attribute once at the beginning of `__calculateAndCacheIntrinsicWidths` and then removing it at the end of the method so that the attribute is toggled as few times as possible.